.travis.yml

```diff
@@ -1,17 +0,0 @@
-language: ruby
```

Comment:
I'd keep `.travis.yml` if it doesn't break anything for you. When you fork a gem I think it is beneficial to keep your version as close as possible to the original so it is both easier for you to upgrade your fork if the original gem gets an update as well as for you to contribute to the original gem by creating a PR against it if you choose to do so

Reply:
Done